dvzo/settings.py

```diff
@@ -54,8 +42,11 @@
     'django.contrib.auth.middleware.AuthenticationMiddleware',
     'django.contrib.messages.middleware.MessageMiddleware',
     'django.middleware.clickjacking.XFrameOptionsMiddleware',
+    'whitenoise.middleware.WhiteNoiseMiddleware',
```

Comment:
This should be moved to second place according to the docs (after sec. middleware).

Reply:
Has been resolved.